src/js/script.js

```diff
@@ -369,38 +371,77 @@
       thisCart.dom.wrapper = element;
       thisCart.dom.toggleTrigger = thisCart.dom.wrapper.querySelector(select.cart.toggleTrigger);
       thisCart.dom.productList = thisCart.dom.wrapper.querySelector(select.cart.productList);
+      thisCart.dom.deliveryFee = thisCart.dom.wrapper.querySelector(select.cart.deliveryFee);
+      thisCart.dom.subtotalPrice = thisCart.dom.wrapper.querySelector(select.cart.subtotalPrice);
+      thisCart.dom.totalPrice = thisCart.dom.wrapper.querySelector(select.cart.totalPrice);
+      thisCart.dom.totalNumber = thisCart.dom.wrapper.querySelector(select.cart.totalNumber);
     }
 
     initActions(){
       const thisCart = this;
       thisCart.dom.toggleTrigger.addEventListener('click', function(){
         thisCart.dom.wrapper.classList.toggle(classNames.cart.wrapperActive);
       });
-
+      thisCart.dom.productList.addEventListener('updated', function() {
+        thisCart.update();
+      });
     }
 
     add(menuProduct){
       const thisCart = this;
-      console.log('adding product', menuProduct);
+      //console.log('adding product', menuProduct);
   
       /*generare HTML based on template */
       const generateHTML = templates.cartProduct(menuProduct);
   
       /*create element using utils.createElementFromHTML */
-      menuProduct.element = utils.createDOMFromHTML(generateHTML);
+      const generatedDOM = utils.createDOMFromHTML(generatedHTML);
    
       /*add elemnt to menu */
       thisCart.dom.productList.appendChild(menuProduct.element);
```

Comment:
2. Tutaj powinnaś do metody `appendChild` przekazać `generatedDom`, a nie `menu Product.element`
```js
thisCart.dom.productList.appendChild(generatedDOM)
```
To jedna z poprawek, które Ci przesłałem w komentarzach do poprzedniego zadania.